orca-queue/src/main/kotlin/com/netflix/spinnaker/orca/q/handler/StartExecutionHandler.kt

```diff
@@ -34,15 +36,19 @@ class StartExecutionHandler(
 
   override fun handle(message: StartExecution) {
     message.withExecution { execution ->
-      repository.updateStatus(message.executionId, RUNNING)
+      if (execution.status == CANCELED || execution.isCanceled) {
```

Comment:
Worth checking any other status ... the execution should really be NOT_STARTED?

Reply:
Yup I agree, I had that thought last night - changing!